packages/ipfs-core-utils/src/files/normalise-input/utils.js

```diff
@@ -15,7 +13,7 @@ function isBytes (obj) {
  * @returns {obj is Blob}
```

Comment:
Because the `const { Blob } = ...` line was removed, this is now the global `Blob` so TS will be unhappy if someone explicitly passes `@web-std/file` objects in your code.

Might just need to remove the `@returns` annotation so tsc will derive a boolean return type.

Reply:
return the `@returns` causes typescript to also not be happy, I added globalThis to the type